corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html

```diff
@@ -116,6 +116,16 @@
 
 {% include 'app_manager/v1/partials/case_list_filtering.html' %}
 
+{{ request|toggle_tag_info:"DETAIL_LIST_TAB_NODESETS" }}
+{% if request|toggle_enabled:'DETAIL_LIST_TAB_NODESETS' %}
+<div data-bind="with: shortScreen">
+    <input data-bind="checked: sortNodesetColumns" type="checkbox">
+    <span>
+        {% trans "Sort entries in the Nodeset tabs" %}
```

Comment:
This would also be a good place for a help icon, to remind people how this works (that it sorts by the first column, and in what order it sorts).